gateway-ha/pom.xml

```diff
@@ -42,6 +42,11 @@
             <artifactId>jackson-core</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>jsr305</artifactId>
```

Comment:
It's fine for temporary change, but we should remove this dependency eventually. https://github.com/trinodb/trino/pull/18047

Reply:
Updated to PR .. was a quick pivot ;-)